### DISCUSSION TITLE
option value cmd-completion like vim

### DISCUSSION THREAD

**Discussion by Song-Tianxiang** [neovim/neovim#27144] [Q&A] [answered]
In vim, if I enter `:set background=<Tab>`, I will get 
![Screen Shot 2024-01-23 at 3 27 32 PM](https://github.com/neovim/neovim/assets/149415622/48ba2b10-cb81-4388-8f14-9b36b07b1c94)
But in neovim, I will get 
![Screen Shot 2024-01-23 at 3 27 55 PM](https://github.com/neovim/neovim/assets/149415622/aa1b31ef-69c6-4e17-b69b-0312980cf780)
How can I get option completion  like vim in neovim?

I can get option completion but not option value completion in neovim.
In neovim, enter `:set back<Tab>` I can get 
![Screen Shot 2024-01-23 at 3 33 07 PM](https://github.com/neovim/neovim/assets/149415622/e17f4fa9-02c4-4846-9fdf-5b7b1a0645d7)
But I can not get `background` vaule compltion (dark and light) as above.

**Comment by zeertzjq** [comment, marked as answer]
Use nightly version.
